lib/oban/queue/executor.ex

```diff
@@ -59,29 +85,28 @@ defmodule Oban.Queue.Executor do
 
   # Helpers
 
-  defp to_module(worker) when is_binary(worker) do
-    worker
-    |> String.split(".")
-    |> Module.safe_concat()
+  defp worker_module(%Job{worker: worker} = job) when is_binary(worker) do
```

Comment:
I refactored `to_module` into this function because it was called in several places and each had to rescue the possible exception from `Module.safe_concat()`. This way we can format that error in one place, using it when necessary and ignoring it when we want (as in backoff)

Reply:
Nicely refactored 👍